Jenkinsfile

```diff
@@ -185,7 +189,7 @@ pipeline {
                     try {
                         deleteDir()
                         unstash "CIScripts"
-                        // powershell script: './CIScripts/Test.ps1'
+                        powershell script: './CIScripts/Test.ps1'
```

Comment:
While we're at it, I think we can remove TestIntegration.ps1 file from the repo.